package.json

```diff
@@ -21,11 +21,12 @@
   "dependencies": {
     "addressit": "git://github.com/dianashk/addressit.git#temp",
     "extend": "^3.0.0",
-    "lodash": "^4.11.1",
+    "lodash": "4.14.0",
+    "node-postal": "openvenues/node-postal",
```

Comment:
For the version I was actually refereeing to the node-postal module, but locking loofah down is good too. 😀 Sorry I should've been more specific.